communication/slack-config/usergroups.yaml

```diff
@@ -246,21 +246,22 @@ usergroups:
     description: Members of the Cluster API Release Team
     members:
       - adilGhaffarDev
-      - aniruddha2000
+      - cahillsf
       - CecileRobertMichon
```

Comment:
same question here as the github groups, should we remove @sbueringer and I since @vincepri is the maintainer for this release

Reply:
Fine for me. Same question for yourself? :)

Reply:
Sounds good to me

Reply:
> same question here as the github groups, should we remove @sbueringer and I since @vincepri is the maintainer for this release

Done